plugins/outputs/elasticsearch/elasticsearch_test.go

```diff
@@ -21,6 +21,7 @@ func TestConnectAndWriteIntegration(t *testing.T) {
 	e := &Elasticsearch{
 		URLs:                urls,
 		IndexName:           "test-%Y.%m.%d",
+		EnableGzip:          true,
```

Comment:
Please don't modify existing tests but rather add a new one with gzip enabled.

Reply:
@srebhan 
Sure.
Any suggestions on how to setup the test to verify gzip actually enabled?
I captured the pcap from both setup and put it in the description.
As in writing unit test, all I can think of is to make requests with gzip disabled/enabled and expect no error from requests. Suggestion appreciated.

Reply:
I think it would be easiest to setup a http-test-server and check if the submitted content is a valid gzip...

Reply:
@srebhan Added tests.